e2e/suite1/specs/trace-view-scrolling.ts

```diff
@@ -0,0 +1,34 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
+  describeName: 'Trace view',
+  itName: 'Can lazy load big traces',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
```

Comment:
Just a small nit, you don't need to use e2e.scenario in this case as you're not adding a dash or data source. Could be easier to just do:
```ts
describe('Trace view', () => {
  it('Can lazy load big traces', () => {
    e2e.flows.login('admin', 'admin');
```